projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/publicize-connection-test-results.php

```diff
@@ -122,7 +125,7 @@ public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAna
 
 		$response = rest_ensure_response( $items );
 
-		$response->header( 'X-WP-Total', count( $items ) );
+		$response->header( 'X-WP-Total', is_countable( $items ) ? count( $items ) : 0 );
```

Comment:
This seems kind of strange, considering we just iterated over the array above. Plus it gets it wrong, everything except null should be 1, not 0.

Also, I note that PHP doesn't provide `is_countable()` until 7.3. But within WordPress that's ok because WordPress polyfills it. OTOH, let's hope none of this changed code (like the markdown lib below) is ever called where WordPress hasn't done the polyfill...

Reply:
nod, considering a false positive.